### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by jsoref**
#### What is the purpose of this change?

grammar -- the eventual goal is making it easier to read through the `rclone config` phase

#### Was the change discussed in an issue or in the forum before?

I mentioned it offhand in #4679, but I wouldn't say it was discussed

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [ ] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/s3.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ name> remote
 Type of storage to configure.
 Choose a number from below, or type in your own value
 [snip]
-XX / Amazon S3 Compliant Storage Providers (AWS, Ceph, Dreamhost, IBM COS, Minio)
+XX / Amazon S3 Compliant Storage Providers including AWS, Ceph, Dreamhost, IBM COS, Minio, and Tencent COS
```
Comment (jsoref) [inline docs/content/s3.md:66]: This feels like a place where the file got out of sync with the rest of the codebase.

**Comment by jsoref** [comment]
Fwiw, these corrections were partially guided by feeding a transcript of using `rclone config` to https://app.grammarly.com -- In theory I know when to use a `-` between words, but I'm quite happy to let a tool tell me :-).

**Comment by jsoref** [comment]
homebrew seems to be flaking out

**Comment by edwardxml** [comment]
These are handy changes

**Comment by ncw** [comment]
Thanks for doing this! Apologies for the delay in reviewing.

I've had a look at the changes and they look good to me :-)

I've rebased this, and renamed the commits to start with `docs:` - hopefully the build will go green this time.

Once it does - I'll merge - thank you :-)

**Comment by ncw** [comment]
All looks good - I'll merge now :-)

**Pull request merged**